lib/engine/step/g_18_co/mine_income.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
```

Comment:
I am unsure if adding a new step for adding the mine income to the treasury ( post running trains ) is the proper solution as there is no real user interaction. The alternative would seem to be to add that logic to the dividend step.

Reply:
correct, you don't need a step, just do it in the dividend